pkg/operator/ceph/csi/spec.go

```diff
@@ -229,6 +230,14 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		}
 	}
 
+	csiPluginImage := strings.Split(CSIParam.CSIPluginImage, ":")
+	// as ceph-csi v2.x.x support resizer, enable it
+	if len(csiPluginImage) > 1 {
+		if !strings.HasPrefix(csiPluginImage[1], "v1.2.") {
```

Comment:
Why not use `strings.HasPrefix(csiPluginImage[1], "v2.")`? This seems more intuitive.

Reply:
I have to change the template to `{{ if not .EnableResizer }}`

Reply:
yes i can do it